tests/Unit/Delivery/AssetTest.php

```diff
@@ -85,6 +91,42 @@ public function testGetter()
         $this->assertEquals(new \DateTimeImmutable('2013-09-02T14:56:34.240Z'), $asset->getUpdatedAt());
     }
 
+    public function testGetTitleWithLocale()
+    {
+        $asset = $this->asset;
+
+        $this->assertEquals('Nyan Cat', $asset->getTitle());
+        $this->assertEquals('Nyan Cat', $asset->getTitle('en-US'));
+        $this->assertEquals('Nyan Cat', $asset->getTitle('tlh'));
```

Comment:
should we add a test case that has a different expected value other than `'Nyan Cat'` i.e. a translated title?